{
  "dialog.button.Cancel" : "Cancel",
  "dialog.button.Continue" : "Continue",
  "dialog.button.Exit" : "Exit",
  "dialog.button.OK" : "OK",
  "dialog.button.Reset" : "Reset",
  "dialog.button.Retry" : "Retry",
  "dialog.title.error" : "Error",
  "dialog.title.warning" : "Warning",
  "emptyComponentMessage" : "No Results",
  "event.Back" : "Back",
  "event.Home" : "Home",
  "loginDialog.title" : "Login",
  "notification.activityProgress" : "Loading...",
  "notification.blobOpenError" : "This file cannot be opened.",
  "notification.genericError" : "An error occurred.",
  "notification.loginProgress" : "Authenticating...",
  "notification.synchronizationError" : "Data synchronization failed.",
  "notification.synchronizationForbidden" : "Access denied: please login again.",
  "notification.synchronizationNetworkError" : "Problems connecting to the network. Please try again later.",
  "notification.synchronizationNetworkUnavailable" : "Network unavailable. Please check your network connection and try again.",
  "notification.synchronizationProgress" : "Synchronizing data...",
  "notification.updateError" : "Update failed due to internal error.",
  "notification.updateProgress" : "Updating...",
  "typeValidation.date" : "the value must be a valid date",
  "typeValidation.decimal" : "the value must be a valid number",
  "typeValidation.float" : "the value must be a valid float",
  "typeValidation.integer" : "the value must be a valid integer",
  "typeValidation.time" : "the value must be a valid time",
  "typeValidation.timestamp" : "the value must be a valid timestamp",
  "update.loginDialog.message" : "Please enter your credentials to continue with the update.",
  "update.networkDialog.message" : "Network connectivity is required for performing this update. Please check your network and try again.",
  "update.resetDialog.message" : "Some local changes are yet to be synchronized with the server. Resetting will cause all changes to be lost."
}